AzureCalling/app/src/main/res/layout-land/activity_call.xml

```diff
@@ -117,7 +117,7 @@
         android:id="@+id/call_hangup_overlay"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:background="#66000000"
+        android:background="#CC000000"
```

Comment:
Does it look good on the night mode?
Did you consider defining this color at values/colors.xml with a night mode?

Reply:
Great, thanks Inderpal